chatbot/OpenAI-API-key/OpenAI_API_key.txt

```diff
@@ -0,0 +1 @@
+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
```

Comment:
You should create a secret for this, export it as a system env variable while Github action building, and use it as env var in the script. I can show you how to do this later. It is ok for now.

Reply:
I understand your concern. I will clear the API key for security. In fact, whenever GitHub detects OpenAI's API Key uploaded, it will send a request to OpenAI to invalidate the API Key.